elrond-wasm/src/storage/mappers/linked_list_mapper.rs

```diff
@@ -0,0 +1,240 @@
+use super::StorageMapper;
+use crate::api::{ErrorApi, StorageReadApi, StorageWriteApi};
+use crate::storage::{storage_get, storage_set};
+use crate::types::{BoxedBytes};
+use core::marker::PhantomData;
+use elrond_codec::{TopDecode, TopEncode};
+use elrond_codec::elrond_codec_derive::{TopDecode, TopEncode};
+
+const NULL_ENTRY : u32 = 0;
+const LEN_IDENTIFIER : &[u8] = b".len";
+const NEW_KEY_IDENTIFIER : &[u8] = b".new";
+const FRONT_IDENTIFIER : &[u8] = b".front";
+const BACK_IDENTIFIER : &[u8] = b".back";
+const NODE_IDENTIFIER : &[u8] = b"*";
+const VALUE_IDENTIFIER : &[u8] = b"&";
```

Comment:
Please also add a simple mandos test to make the keys obvious.
You can add a method and the test in basic-features.

Reply:
I added it.